ADR/ADR044-translate-api-response-at-request-time.md

```diff
@@ -0,0 +1,417 @@
+
+```jsonc
+{
+  "form_id": "7",
+  "name": "A form for testing file upload",
+  "submission_email": "david.biddle@digital.cabinet-office.gov.uk",
+  "privacy_policy_url": null,
+  "form_slug": "a-form-for-testing-file-upload",
+  "support_email": null,
+  "support_phone": null,
+  "support_url": null,
+  "support_url_text": null,
+  "declaration_text": null,
+  "question_section_completed": false,
+  "declaration_section_completed": false,
+  "created_at": "2025-03-27T12:05:49.395300Z",
+  "updated_at": "2025-07-29T14:35:44.423406Z",
+  "creator_id": 1,
+  "what_happens_next_markdown": null,
+  "payment_url": null,
+  "submission_type": "email",
+  "share_preview_completed": false,
+  "s3_bucket_name": null,
+  "s3_bucket_aws_account_id": null,
+  "s3_bucket_region": null,
+  "language": "en",
+  "start_page": 33,
+  "steps": [
+    {
+      "id": 33,
+      "position": 1,
+      "next_step_id": null,
+      "type": "question_page",
+      "data": {
+        "question_text": "Upload your file",
+        "hint_text": "",
+        "answer_type": "file",
+        "is_optional": false,
+        "answer_settings": null,
+        "page_heading": "Annual report",
+        "guidance_markdown": "Some guidance",
+        "is_repeatable": false
+      },
+      "routing_conditions": []
+    }
+  ]
+}
+```
+
+And `/forms/7/live?locale=cy`, would return the same object with any Welsh translations overwriting the English:
+
+```jsonc
+{
+  "form_id": "7",
+  "name": "Ffurflen ar gyfer profi uwchlwytho ffeiliau",
+  "submission_email": "david.biddle@digital.cabinet-office.gov.uk",
+  "privacy_policy_url": null,
+  "form_slug": "a-form-for-testing-file-upload",
+  "support_email": null,
+  "support_phone": null,
+  "support_url": null,
+  "support_url_text": null,
+  "declaration_text": null,
+  "question_section_completed": false,
+  "declaration_section_completed": false,
+  "created_at": "2025-03-27T12:05:49.395300Z",
+  "updated_at": "2025-07-29T14:35:44.423406Z",
+  "creator_id": 1,
+  "what_happens_next_markdown": null,
+  "payment_url": null,
+  "submission_type": "email",
+  "share_preview_completed": false,
+  "s3_bucket_name": null,
+  "s3_bucket_aws_account_id": null,
+  "s3_bucket_region": null,
+  "language": "en",
+  "start_page": 33,
+  "steps": [
+    {
+      "id": 33,
+      "position": 1,
+      "next_step_id": null,
+      "type": "question_page",
+      "data": {
+        "question_text": "Llwythwch eich ffeil i fyny",
+        "hint_text": "",
+        "answer_type": "file",
+        "is_optional": false,
+        "answer_settings": null,
+        "page_heading": "Adroddiad blynyddol",
+        "guidance_markdown": "Peth arweiniad",
+        "is_repeatable": false
+      },
+      "routing_conditions": []
+    }
+  ]
+}
+```
+
+Note that in this case we would still need to store the individual translations in the made_live_form JSON. We could use one of the formats described in option 1 for this.
```

Comment:
As we're dealing with the v2 API here, we should talk about form documents rather than made_live_forms

```suggestion
Note that in this case we would still need to store the individual translations in the form document JSON. We could use one of the formats described in option 1 for this.
```